src/utils/popup-keyboard-accessibility.tsx

```diff
@@ -89,9 +89,11 @@ export const withKeyboardA11y = (WrappedComponent): any =>
      * @memberof HOC
      */
     componentWillUnmount(): void {
-      if (this._previouslyActiveElement) {
-        this._previouslyActiveElement.focus();
-      }
+        setTimeout(() => {
```

Comment:
you can use
https://github.com/kaltura/playkit-js-ui/blob/28b456aca775250dcef0d100d86419b09eada209/src/utils/focus-element.ts#L9
to verify that the element is focused
that way you don't have to "gamble" that the timeout that you selected will work in all cases